iib/workers/tasks/build.py

```diff
@@ -123,7 +123,7 @@ def _create_and_push_manifest_list(request_id, arches):
     return output_pull_spec
 
 
-def _finish_request_post_build(
+def _check_to_modify_index_image(
```

Comment:
Docstring should also reflect this change, more specifically, it shouldn't say "Finish the request". It should also indicate that it has a return value.

How about calling it something like `update_index_image_in_request` to better reflect its intent?

Reply:
Yup, let me add those in the next commit. TBH I wasn't happy with '_check_to_modify_index_image' either